PortalApp/index.html

```diff
@@ -81,6 +81,9 @@
     <script type="text/javascript" src="//maps.googleapis.com/maps/api/js?key=AIzaSyDYHJjsVAniNFUxSZnIO6pyVBxBqRt9gmw&sensor=false"></script>
     <!--script type="text/javascript" src="//maps.google.com/maps/api/js?sensor=false"></script-->
 
+    <!-- Google Tag Manager -->
```

Comment:
It looks like the same changes are made on #76

https://github.com/specify/webportal-installer/blob/856d6391581ca6fe7eb6a33eeef142a6abebf4ae/PortalApp/index.html#L84-L85

Reply:
If this was intentional, it may be worth noting that the Pull Requests are now 'out of sync': there is an extra commit in https://github.com/specify/webportal-installer/pull/76 which is not present in this PR: https://github.com/specify/webportal-installer/pull/76/commits/db2e7f23c008051cb41a23146af46eb29b99c339